index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <!-- css -->
+    <link rel="stylesheet" href="styles.css" />
+    <!-- fontAwesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <!-- fonts -->
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap");
+    </style>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <div class="header">
+          <h2 class="header-title">Welcome</h2>
+          <div class="hamburger">
+            <button class="sidebar-toggle">
+              <i class="fas fa-bars"></i>
+            </button>
+          </div>
+        </div>
+      </header>
+
+      <!-- main -->
+      <main>
+        <div class="main-title">
+          <h1 class="heading">Hey there. I'm Maxwell</h1>
+          <h1 class="heading-two">I'm a developer</h1>
+        </div>
+        <p>
+          I can help you build a product , feature or website look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>
+
+        <!-- Social Media Links -->
+        <div class="social-media-container">
+          <ul>
+            <li>
+              <a href="https://@Maxwell_Chuks67"
+                ><img src="./images/Twitter.png" alt="Twitter Logo"
+              /></a>
+              <a href="https://www.linkedin.com/in/chukwuemeka-orji-50b766241/"
+                ><img src="./images/Linkedin.png" alt="Linkedin Logo"
+              /></a>
+              <a href="https://github.com/Maxwell011"
+                ><img src="./images/Medium.png" alt="Medium Logo"
+              /></a>
+              <a href="https://github.com/Maxwell011"
+                ><img src="./images/Vector.png" alt="GitHub Logo"
+              /></a>
+              <a href="https://github.com/Maxwell011"
+                ><img src="./images/Angelist.png" alt="Angelist Logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
+
+        <!-- Disabled -->
+        <div>
+          <div class="disabled-container">
+            <img
+              src="./images/Disabled.png"
+              alt="Arrow Down"
+              class="disabled"
+            />
```

Comment:
- [ ] This should be bigger, kindly update it accordingly to the design
![image](https://user-images.githubusercontent.com/62705947/185248663-a58b91d9-bb9d-46c8-88f2-1f13cfad6880.png)